zulipterminal/ui_tools/views.py

```diff
@@ -1002,14 +1003,19 @@ def __init__(self, controller: Any, color: str,
 
 
 class MsgInfoView(PopUpView):
-    def __init__(self, controller: Any, msg: Message, title: str) -> None:
+    def __init__(self, controller: Any, msg: Message, title: str,
+                 message_links: 'OrderedDict[str, Tuple[str, int, bool]]',
```

Comment:
If this can be None, then `Optional[MESSAGE_LINKS]`?

Reply:
This can be an empty `OrderedDict` but I believe mypy covers that currently?